src/sdk/model/org/__json__/org-vdc-network-json.ts

```diff
@@ -0,0 +1,29 @@
+import { IpRangeJson } from '../../common/ip-range/__json__/ip-range-json';
+import { FenceModeType } from '../../common/__json__/fence-mode-type';
+
+/**
+ * Org Vdc Network JSON interface.
+ */
+export interface OrgVdcNetworkJson {
```

Comment:
We already have this and the corresponding wrapper class, its called `InternalNetwork` in here